index.html

```diff
@@ -9,45 +9,217 @@
-</p>
+    <section class="container">
+            <h1 class="salutation">I’m Austin. <br> Glad to see you!</h1>
 
+            <p class="developer"> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
 
-<footer>
-<h2 class="connect">LET'S CONNECT</h2>
+    <h2 class="connect">LET'S CONNECT</h2>
 
-<section class="social">
-    <ul >
+        <section class="social">
+            <ul >
+                <li><a href="https://twitter.com/home"><img src="images/twitter.jpg" alt="twitter"></a></li>
+                <li> <a href="https://www.linkedin.com/feed/"><img src="images/Linkedin-icon.jpg" alt="Linkedin"></a></li>
+                <li><a href="https://twitter.com/home"><img src="images/Vector.jpg" alt="twitter"></a></li>
+                <li><a href="https://github.com/ausiemass/portfolio"><img src="images/github.jpg" alt="GITHUB"></a></li>
+                <li><a href="https://twitter.com/home"><img src="images/Vector (2).jpg" alt="twitter"></a></li>
+            </ul> 
+        </section>  
+    </section>
+    <!-- this section present some of the projects done -->
+
+
+    
+
+    <section class="works">
+        <!-- tonic -->
+        <div class="grid-container">
+            <div class="grid-item"><img src="images/tonic1.jpg" alt="works 1"></div>
+            <div class="grid-item">
+                <div class="works1">
+                    <h1 class="salutation">Tonic</h1>
+                    <div class="flexf">
+                        <p class="canopy">canopy</p> 
+                        <img src="images/Counter.jpg" alt="counter">
+                        <p class="back">back end dev</p>
+                        <img src="images/Counter.jpg" alt="counter">
+                        <p class="back">2015</p>
+                    </div>
+                </div>
+                <div>
+                    <p class="developer">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              
+                </div>
+                <div>
+                    <ul>
+                        <li><button class="btn1">HTML</button></li>
+                        <li><button class="btn1">CSS</button></li>
+                        <li><button class="btn1">Javascript</button></li>
+                    </ul>
+                </div>
+                <button class="resume">See Project</button>
+            </div>
+            
+          </div>
+          <!-- end -->
+
+        <!-- Multi-Post Stories -->
+        <div class="grid-container">
+            <div class="grid-item"><img src="images/Snapshoot Portfolio.png" alt="works 1"></div>
+            <div class="grid-item">
+                <div class="works1">
+                    <h1 class="salutation">Multi-Post
+                        Stories</h1>
```

Comment:
- [ ] I will suggest that you avoid using multiple `h1 tags` in your project :+1:  Using multiple `<h1> tags` in a project is generally not recommended because it can lead to confusion and semantic inconsistency within the HTML structure. The purpose of the `<h1> tag` is to represent the main heading or the most important heading on a web page. It carries significant weight in terms of search engine optimization (SEO) and accessibility. :100: 
- [ ] In your case, this text **I’m Austin. <br> Glad to see you!**  in your `Headline section` is serving as your most important heading. So kindly use the other `hx tags`. Eg. (`h2`,`h3`,`h4`, etc) for the card headings instead of the `h1 tag` :+1: